src/app/lecture/knowledge-node/learning-objects/arrange-task/arrange-task.component.html

```diff
@@ -1,10 +1,16 @@
 <div id="arrange-task" class="mat-card">
-  <h3 class="mat-h3"><markdown>{{learningObject.text}}</markdown></h3>
-  <div cdkDropListGroup fxLayout="row wrap" fxLayoutAlign="space-between">
+  <div><markdown>{{learningObject.text}}</markdown></div>
+  <div
+    cdkDropListGroup
+    [cdkDropListGroupDisabled]="answered"
+    fxLayout="row wrap"
+    ngClass.xl="m-lg"
+    ngClass.lg="m-lg"
+    ngClass.md="m-md">
     <div
```

Comment:
Is it possible for each container to be tall as the tallest container in the row (something like flex to height of parent)? Looking at this image:
![image](https://user-images.githubusercontent.com/7092212/113821539-83542200-977c-11eb-94cc-57e000f7505b.png)
it would be helpful if there was a more uniform look as this is a bit chaotic.